examples/react/start-clerk-basic/app/routes/_authed.tsx

```diff
@@ -11,7 +11,7 @@ export const Route = createFileRoute('/_authed')({
     if (error.message === 'Not authenticated') {
       return (
         <div className="flex items-center justify-center p-12">
-          <SignIn forceRedirectUrl={window.location.href} />
+          <SignIn routing='hash' forceRedirectUrl={window.location.href} />
```

Comment:
Any reason why `routing="hash"` is required here?

Reply:
Not entirely sure to be honest. After updating clerk version, when _not_ signed in and trying to navigate from /posts -> home the routing is broken and I was getting "not found". Adding routing="hash" fixed the issue for me

Reply:
Hmmm, alright. Could you fix the merge conflicts? I'll try it out before merging.

Reply:
fixed merge conflicts. if you aren't encountering the issue without the "hash" change, or have a better solution please let me know!